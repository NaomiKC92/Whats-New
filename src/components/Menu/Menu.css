* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}

nav {
  /* border: 2px solid red; */
  display: flex;
  flex-direction: column;
  width: 25%;
}

h1 {
  margin-left: 20px;
  font-size: 200%;
  font-weight: bolder;
  color: #FFAE00;
}

.menu-btns {
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  margin: 40px auto auto 20px;
  justify-content: space-between;
  height: 350px;
}

button {
  background-color: #141427;
  color: #F0F0F1;
  font-size: 150%;
  border: none;
}

button:hover {
  background-color: #FFAE00;
  cursor: pointer;
  height: 56px;
  border-radius: 1px 25px 25px 1px;
  width: 100%;
  text-align: left;
}
