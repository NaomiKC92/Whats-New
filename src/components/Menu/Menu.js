import React from 'react';
import './Menu.css'

const Menu = ({switchTopic}) => {
  return (
    <div className='menu-items'>
      <button id='local' onClick={switchTopic}>Local</button>
      <button id='entertainment' onClick={switchTopic}>Entertainment</button>
      <button id='science' onClick={switchTopic}>Science</button>
      <button id='health' onClick={switchTopic}>Health</button>
      <button id='technology' onClick={switchTopic}>Technology</button>
    </div>
  )
}

export default Menu;