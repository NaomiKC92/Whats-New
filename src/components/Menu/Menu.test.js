import React from 'react';
import ReactDOM from 'react-dom';
import Menu from './Menu';
import { shallow } from 'enzyme';

describe('Menu', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Menu />)
    expect(wrapper).toMatchSnapshot()
  })
})