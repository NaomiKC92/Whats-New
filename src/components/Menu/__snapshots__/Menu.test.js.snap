// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should match the snapshot 1`] = `
<nav>
  <h1>
    What's New?
  </h1>
  <div
    className="menu-btns"
  >
    <button
      id="local"
    >
      Local
    </button>
    <button
      id="entertainment"
    >
      Entertainment
    </button>
    <button
      id="science"
    >
      Science
    </button>
    <button
      id="health"
    >
      Health
    </button>
    <button
      id="technology"
    >
      Technology
    </button>
  </div>
</nav>
`;
